### PR TITLE
feat: lr_scheduler

### PR DESCRIPTION
**Pull request by SunCreation**
I should've done it earlier.. Sorry I'm so late!
With the lr scheduler, It's much better to see the learning aspect according to lr.

Looking for the best lr, I'll shorten the time!

### CODE REVIEW

**Review by YejinHwang909 — APPROVED** [review]
LGTM~!! 👍
